features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/processor/RecordedDataProcessor.kt

```diff
@@ -50,6 +54,25 @@ internal class RecordedDataProcessor(
         )
     }
 
+    @WorkerThread
+    override fun processWebViewRecord(item: WebViewRecordedDataQueueItem) {
+        JsonParser.parseString(item.serializedRecord)?.asJsonObject?.let {
+            val timestampOffset = timeProvider.getTimestampOffset()
+            it.get("timestamp")?.let { timestamp ->
+                val correctedTimestamp = it.asLong + timestampOffset
+                it.addProperty("timestamp", correctedTimestamp)
+            }
+            val rumContext = item.recordedQueuedItemContext.newRumContext
+            val enrichedRecord = EnrichedRecord(
+                rumContext.applicationId,
+                rumContext.sessionId,
+                rumContext.viewId,
```

Comment:
question/ What's the exact reasoning behind enriching browser record with mobile view ID 🤔? Browser SR records get created with browser's view ID and here we replace it.

Reply:
that was the first approach...now we are going to enrich it with the browser view id in the last PR.